Docker/securityserver/README.md

```diff
@@ -12,45 +12,88 @@ The installed security server is in uninitialized state.
 Admin UI credentials: xrd/secret
 
 ## Running
-```
-# Simple example 
+
+```shell
+# Simple example
 docker run --name my-ss niis/xroad-security-server
 
+# Simple example passing in the XROAD_AUTOLOGIN_PIN for autologin
+docker run --name my-ss niis/xroad-security-server -e XROAD_AUTOLOGIN_PIN="1234"
```

Comment:
This should probably be:
```suggestion
docker run --name my-ss -e XROAD_AUTOLOGIN_PIN="1234" niis/xroad-security-server 
```
There is a similar problem in the other examples as well.